app/components/maintenance-banner/component.js

```diff
@@ -34,6 +45,8 @@ export default Ember.Component.extend({
                             hidden: false,
                             maintenance
                         });
+                        this.set('_info', false);
+                        this.set(`_${levelMap[maintenance.level]}`, true);
```

Comment:
Should gracefully handle the case when maintenance.level is `null`, `undefined`, or not mapped in levelMap.